layers/+spacemacs/spacemacs-defaults/packages.el

```diff
@@ -209,8 +209,10 @@
      ediff-merge-split-window-function 'split-window-horizontally)
     :config
     ;; show org ediffs unfolded
-    (require 'outline)
-    (add-hook 'ediff-prepare-buffer-hook #'outline-show-all)
+    (add-hook 'ediff-prepare-buffer-hook
+              #'(lambda ()
```

Comment:
Generally it is better to define named functions that you add to hooks, rather than adding lambdas.  The former are much easier for users to remove in their configs, if desired.